core/wiki/allfields.tid

```diff
@@ -1,13 +1,43 @@
 title: $:/snippets/allfields
 
-\define renderfield(title)
-<tr class="tc-view-field"><td class="tc-view-field-name">''<$text text=<<__title__>>/>'':</td><td class="tc-view-field-value">//{{$:/language/Docs/Fields/$title$}}//</td></tr>
-\end
 \whitespace trim
+
+\function tf.getLingoText() [[$:/language/Docs/Fields/]] [<title>] +[join[]get[text]]
+
+\procedure renderfield(title)
+<tr class="tc-view-field">
+	<td class="tc-view-field-name">
+		''<$text text=<<title>>/>'':
+	</td>
+	<td class="tc-view-field-value">
+		//<<tf.getLingoText>>//
+	</td>
+	<td class="tc-view-field-links">
+		<!-- show-filter-count expects a filter string. It does no more processing -->
+		<$macrocall $name="show-filter-count" filter=`[has[$(title)$]sort[]]`>>
```

Comment:
The use of textual substitution gives a subtle regression here; it means that renderfield will no longer work with a title that contains a closing square bracket.

Reply:
IMO it can't be a regression, since the "show-filter-count" macro is new to this PR. The problem is, that the "substituted" filter is copied "as is" to the $AdvancedSearch input field, which needs a valid filter string. So if we would use `[has<title>sort[]]` this exact string would be copied into the input, which will not work. 

Eg: an `author]` field can not be witten into the AdvancedSearch at the moment. It would result in `[has[author]]sort[]]` 
At the moment AdvancedSearch can not handle `[has<title>sort[]]` yet.

Reply:
I'll change the function "signature" to: `\procedure renderfield(field)`